addon/components/pop-over/component.js

```diff
@@ -372,6 +388,12 @@ export default Component.extend({
       targetRect.translateX(-1 * $boundingElement.scrollLeft());
     }
 
+    let staticElement = this.getStaticParentOffset(target.element);
+    let staticOffset = { top: 0, left: 0 };
+    if (staticElement) {
+      staticOffset = jQuery(staticElement).offset();
```

Comment:
Instead of using offset here, could we try using `Rectangle.ofElement`? (If that returns the wrong value, make a comment to that effect, since that's a good gotcha to know about!!)